src/main/scala/org/ergoplatform/modifiers/history/PoPowAlgos.scala

```diff
@@ -0,0 +1,74 @@
+package org.ergoplatform.modifiers.history
+
+import org.ergoplatform.mining.difficulty.RequiredDifficulty
+import org.ergoplatform.settings.Constants
+import scorex.util.{ModifierId, bytesToId, idToBytes}
+
+object PoPowAlgos {
+
+  val InterlinksPrefixCode: Byte = 0x01
+
+  @inline def maxLevelOf(header: Header): Int = {
```

Comment:
should be private

Reply:
@catena2w , this will be used inside `NiPoPowProof` in future

Reply:
Let's make it private now and will make it public when NiPoPowProof will be implemented